products/contao.md

```diff
@@ -16,8 +16,8 @@ releases:
     releaseDate: 2022-08-18
```

Comment:
Contao 5.0 is EOL but that seems normal considering https://contao.org/en/release-plan.html.